### ISSUE TITLE
플러그인 활성화 실패

### ISSUE THREAD

**Issue by calvinsnax** [akasima/xero_commerce#95]
![59025032-335d5900-888e-11e9-9945-3d015a8da86f](https://user-images.githubusercontent.com/30161600/59242318-0c17da80-8c46-11e9-8463-fcb2c0a82647.png)
![59025030-322c2c00-888e-11e9-8acf-1276b20b42c8](https://user-images.githubusercontent.com/30161600/59242319-0c17da80-8c46-11e9-9fea-65abb4c23b5e.png)

확인이 안되시는 것 같아서 다시 올려드립니다. 활성화가 실패되며, 디버그 내용은 위와 같습니다.

**Comment by Kazto-lsk** [comment]
실례가 안되신다면 어느 환경에서 해당 문제가 발생했는지 기술 부탁드립니다.
재현하고 알려드리겠습니다.

**Comment by calvinsnax** [comment]
- Linux 64Bit
- Apache 2
- UTF-8
- PHP 7.2
- MariaDB 10.X

iwinv 호스팅 사용 중입니다.

**Comment by Kazto-lsk** [comment]
네, 감사합니다. 동일한 환경에서 설치를 진행해보고 어떻게 하면 해당 문제를 해소할 수 있을지 확인해보겠습니다.

**Comment by Kazto-lsk** [comment]
안녕하세요, 동일한 호스팅에서 동일한 환경으로 세팅한 후 테스트 해 보았습니다.

아마도, 플러그인을 설치하면서 autoload 파일이 갱신되지 않아 생기는 문제 같은데요.
SSH 사용이 가능하다면

xe3_root/plugins/xero_commerce 로 이동하시어 composer install 이후 이용해보시기 바랍니다.

만일 그래도 해결이 되지 않으신다면, 알려주세요.
